lib/environment_config_transmogrifier.rb

```diff
@@ -47,14 +47,28 @@ def self.transmogrify(base_config, environment_templates, secrets: nil)
     base_config
   end
 
-  def self.processMustacheTemplate(value, input_hash)
+  def self.processMustacheTemplate(value, input_hash, key)
+    delimiter = '{{=(( ))=}}'
     begin
-      mustache_value=NoEscapeMustache.render("{{=(( ))=}}#{value}", input_hash)
+      mustache_value=NoEscapeMustache.render("#{delimiter}#{value}", input_hash)
       # replace new lines with double new lines for proper new-line YAML parsing
       mustache_value="#{mustache_value}".gsub("\n", "\n\n")
       YAML.load(mustache_value)
     rescue => e
-      raise LoadYamlFromMustacheError, "Could not load config key '#{key}': #{e.message}"
+      lines = e.message.split(/\n/)
+      if lines.size >= 4 && lines[0]["Illegal content in tag"] && lines[1]["Line "]
```

Comment:
If we do that, we risk exposing a secret, which is why the message fiddling in the first place.